TailorSort.java

```diff
@@ -43,6 +43,36 @@ private static boolean isArraySorted(Integer[] a){
         //WRITE YOUR CODE HERE
         //RETURN THE RESULT COATS ARRAY WITH ALL SORTED
         //YOU HAVE TO DO MINIMUM COMPARISONS, CLUE: USE index parameter in your sorting technique
+    	int i = index;
+    	int j = index - 1;
+    	int n = inputCoats.length;
+    	int temp , temp1;
+    	
+    	int checks =0;
+    	while(i<n){
+    		checks++;
+    		if(inputCoats[i]< inputCoats[i-1]){
```

Comment:
The sorting technique that you chosen was good, but do we need this if condition at all? just inner j loop check is enough?